src/Money.php

```diff
@@ -350,15 +350,20 @@ public function divide($divisor, $roundingMode = self::ROUND_HALF_UP)
     public function allocate(array $ratios)
     {
         if (count($ratios) === 0) {
-            throw new \InvalidArgumentException('Cannot allocate to none');
+            throw new \InvalidArgumentException('Cannot allocate to none, ratios cannot be an empty array');
```

Comment:
@sandrokeil Could you clarify a bit more? Don't get your point. It's the argument that is wrong here, right? It's not a Money exception.